packages/itmat-commons/src/models/job.ts

```diff
@@ -10,7 +10,7 @@ export interface IJobEntry<dataobj> {
     requestTime: number;
     receivedFiles: string[];
     status: string;
-    error: null | object;
+    error: null | Record<string, unknown>;
```

Comment:
Does this really corresponding to the behaviour of the code??

Reply:
This this context yes. `object` is not recommended due to problems in the compiler for key inception: https://github.com/microsoft/TypeScript/issues/21732